onnxscript/tests/functions/onnxfns2_test.py

```diff
@@ -115,6 +115,10 @@ def test_onnxfns_hardmax(self):
     #         mode=default_mode,
     #         skip_test_names=[])
 
+    @unittest.skipIf(
+        Version(onnxruntime.__version__) == Version("1.14"),
```

Comment:
May be `<=` would be better than `==`

Reply:
It was successful in 1.13

Reply:
Hmm I think this is another issue that is not OptionalGetElement.

Reply:
Oh, I see! I guess we need to investigate it then. (Agree the condition above seems appropriate in this case.)